### PR TITLE
feat: add support for aqua satellite

### PR DESCRIPTION
**Pull request by hollandjg**
Add aqua satellite as an option on the load script. Everything else should work as before.

### CODE REVIEW

File: src/ebfloeseg/load.py [hunk 1]
```diff
@@ -44,13 +50,20 @@ def load(
...
-        case ImageType.cloud:
+        case (Satellite.terra, ImageType.cloud):
             layers = "MODIS_Terra_Cloud_Fraction_Day"
-        case ImageType.landmask:
+        case (Satellite.aqua, ImageType.truecolor):
+            layers = "MODIS_Aqua_CorrectedReflectance_TrueColor"
+        case (Satellite.aqua, ImageType.cloud):
+            layers = "MODIS_Aqua_Cloud_Fraction_Day"
+        case (_, ImageType.landmask):
             layers = "OSM_Land_Mask"
```
Comment (cpaniaguam) [inline src/ebfloeseg/load.py:63]: In the spirit of using enums, what about putting these values for the layers in one as well?

**Review by cpaniaguam — COMMENTED** [review]
Thanks. Just one question.

**Review by cpaniaguam — COMMENTED** [review]
Thanks. Just one question.

File: src/ebfloeseg/load.py (hunk 1 above)
Comment (hollandjg) [reply src/ebfloeseg/load.py:63]: I'm not sure. Right now, I can only think of this one place where that Enum would be used, and the enum would be a 1:1 reflection of the structure of this `case`/`match` block. 

As soon as the layer names get used elsewhere in addition to here, I think making it an Enum makes total sense. But for now, I think the simplest way is to leave these as simple strings within this one function.

**Review by cpaniaguam — APPROVED** [review]
Good!

**Pull request merged**